platform/darwin/src/MGLGeometry.h

```diff
@@ -24,6 +24,14 @@ typedef struct __attribute__((objc_boxable)) MGLMapPoint {
     CGFloat zoomLevel;
 } MGLMapPoint;
 
+/* Defines a 4x4 matrix. */
```

Comment:
Unless we’re going to also document all the fields of this struct and the APIs that use it, let’s `:nodoc:` this struct.

Reply:
When I generated the docs locally this didn’t make it in (and it appeared in `undocumented.json`), so presumably this comment format is `nodoc`-ish.